RESYNC.md

```diff
@@ -42,48 +44,38 @@ at the beginning of the commit message.
 
 For the sake of transparency, for every resync process we should update the table in `RESYNC.log.md`. The newest resync should appear in the first row. 
 
-## Release Branch Upstream Resync Strategy
-
-### Key Requirements/Constraints
-
-1. Ensure that all the upstream features/updates related to NUMA resource topology scheduler plugin including test updates are backported to both release-4.10 and release-4.11 branches.
-1. As much as possible, keep the base image and golang version intact (go version 1.16 in release-4.10 and go version 1.18 in release-4.11). This is to ensure changes for Z versions are minimal.
-1. Commit history and authorship MUST be preserved when code is synched with upstream irrespective of whether it is full resync or a few PRs are cherry picked. One way of achieving this is to use `git cherry-pick -x <commit>` as it allows preservation of original commit reference within the cherry-picked commit.
-
-### Preparing the local repo clone
-Clone from a personal fork of openshift-kni/scheduler-plugins via a pushable (ssh) url:
-
-`git clone git@github.com:openshift-kni/scheduler-plugins.git`
-
-Add a remote for upstream and fetch its branches:
-
-`git remote add --fetch upstream https://github.com/kubernetes-sigs/scheduler-plugins`
-
-### Creating a new local branch for the new resync
-
-Branch the target openshift-kni/scheduler master branch to a new resync local branch
+## Stable Branches Resync Strategy: cherry-pick flow
 
-`git checkout <release branch>` e.g. `git checkout release-4.10`
+The cherry pick flow should be used to pull targeted changes from a more recent branch to an older branch that
+upstream didn't backport itself.
 
-`git checkout -b "update-$(git symbolic-ref --short HEAD)-$(date +%Y%m%d)"` e.g. `git checkout -b update-release-4.10-20220707`
-
-### Cherry Pick changes from PRs
+### Steps
 
 Go to the upstream PR you are trying to cherrypick and identify the commit hashes.
 
 `git cherry-pick -x <start-commit-hash>^..<end-commit-hash>`
 
-NOTE: -x flag above is used to preserve original commit reference
+NOTE: -x flag above is used to preserve original commit reference.
+**We must always use it to preserve proper reference**
 
-Fix conflicts introduced by kni-local changes and send PR for review.
+Fix conflicts introduced by KNI-local changes and send PR for review.
+Since cherry-picking always mutates the original commit hash (see Appendix B below), cherry-picks must
+have the `[KNI]` prefix tag.
 
```

Comment:
my only concern is how would we be able to differentiate between old-way KNI (only changes specific to the KNI repo) and the cherrypicks that may come from different resources: u/s repo or from a higher branch in the forked repo. I am leaning more towards having a different prefix tag that would help us identify such cherry picks namely from u/s. so if the cherry-pick is already tagged [KNI] do nothing, but if it is coming from u/s then tag that  with a different tag, perhaps [UPSTREAM-CP] as for upstream cherry-pick or something similar

Reply:
adding more tags is fine to me. I'd argue that the source of cherry-picks on a given stable branch can only be
1. upstream repo: we have `[KNI]` + `-x`
2. downstream repo, more recent branch: we have `[KNI]`
so the above should be sufficient. But adding more tags to disambiguate won't hurt. I will update the doc.

Reply:
-x is more helpful if we expand the commit to learn more about it, while listing the git commits in the UI won't give us this hint easily unless the title points it out or we expand it.

Reply:
thanks for the clarifications in the new version, I believe the additional [upstream] will enable us to identify commits that were selected from u/s and not under resync (upstream merge - `git merge`) flow.